plugins/linear-genome-view/src/LinearGenomeView/index.ts

```diff
@@ -1279,6 +1279,9 @@ export function stateModelFactory(pluginManager: PluginManager) {
       // this "clears the view" and makes the view return to the import form
       clearView() {
         self.setDisplayedRegions([])
+
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        self.tracks = [] as any
```

Comment:
the right thing to do here is probably to empy the mobx tracks array. mobx arrays have a `clear` method, see https://mobx.js.org/api.html#observablearray